### PR TITLE
Simplify project setup for android

### LINKED ISSUE #22: Android: Simplify project setup by removing the '.debug' app ID suffix
Currently developers need to setup two Firebase projects/apps to use the app.

- One for Release
- One for Debug, with the `.debug` suffix

I'd like to remove the `.debug` suffix from the `build.gradle` and so make it possible to use only one Firebase project/app (both for release and debug).

Also the `android/README.md` needs to be updated.

### PR DESCRIPTION
**Pull request by sduduzog**
resolves #22

### CODE REVIEW

File: android/app/build.gradle [hunk 1]
```diff
@@ -64,12 +64,11 @@ android {
     buildTypes {
         debug {
             // app id will be "de.janoodle.circlesApp.debug"
-            applicationIdSuffix ".debug"
         }
```
Comment (miquelbeltran) [inline android/app/build.gradle]: 🔍  the whole `debug` section could be removed

File: android/app/build.gradle [hunk 2]
```diff
@@ -64,12 +64,11 @@ android {
...
         release {
             // NOTE: ProGuard is not enabled at the moment
             // See: https://flutter.dev/docs/deployment/android to enable it
-            signingConfig signingConfigs.debug
+            // signingConfig signingConfigs.release
```
Comment (miquelbeltran) [inline android/app/build.gradle]: ❓ why was this necessary?

Comment (sduduzog) [reply android/app/build.gradle]: No signing configurations for a debug build but there is one for a release build. From line 55
```
//     signingConfigs {
//         release {
//             keyAlias keystoreProperties['keyAlias']
//             keyPassword keystoreProperties['keyPassword']
//             storeFile file(keystoreProperties['storeFile'])
//             storePassword keystoreProperties['storePassword']
//         }
//     }
```
Should this be ever required, one would simply uncomment and carry on

Comment (miquelbeltran) [reply android/app/build.gradle]: I think that `signingConfigs.debug` is the default one, I would prefer to keep that one.

That's what the new Flutter project template uses.

I'd prefer to restore back the `signingConfig signingConfigs.debug` then.

**Comment by sduduzog** [comment]
I've been enjoying the journey. I've learnt a couple of things along the way too. I kid you not, before this project I never knew I could switch channels in flutter

**Comment by miquelbeltran** [comment]
I am happy to hear that! Thanks a lot for your works :)
